pkg/mappings/resource-mappings.go

```diff
@@ -0,0 +1,31 @@
+package mappings
+
+import (
+	"context"
+
+	v2vv1alpha1 "github.com/kubevirt/vm-import-operator/pkg/apis/v2v/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// ResourceMappingsFinder provides functionality of retrieving Resource Mapping CRs
+type ResourceMappingsFinder struct {
+	client client.Client
+}
+
+// NewResourceMappingsFinder creates new ResourceMappingsFinder configured with given client
+func NewResourceMappingsFinder(client client.Client) ResourceMappingsFinder {
```

Comment:
It seems like we are not using `NewResourceMappingsFinder`. Was that intended?

Reply:
No, good catch - that's a bug.